dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM rust
```

Comment:
In this commit you should also replace the `docker pull docker.io/wszdexdrf/ledger_emulator` in `.github/workflows/main.yml` with using this Dockerfile

Also, all the CI files should be put in a `ci/` directory

Reply:
Wouldn't that make it considerably slower if it has to build the image every time? The image being pulled is build from the Dockerfile, and since the image doesn't change between runs so I thought building it beforehand and pulling it would save time...